ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +405,184 @@ def test_trustdomain_disable_disables_subdomain(self):
+        ])
+
+        master.put_file_contents('/tmp/user_ssh_priv_key', privkey)
+        master.run_command(['chmod', '600', '/tmp/user_ssh_priv_key'])
+
+        # add group groupa
+        cmd_output = master.run_command(['ipa', 'group-add', 'groupa'])
+        assert 'Added group "groupa"' in cmd_output.stdout_text
+
+        # add group groupb
+        cmd_output = master.run_command(['ipa', 'group-add', 'groupb'])
+        assert 'Added group "groupb"' in cmd_output.stdout_text
+
+        # add group groupc
+        cmd_output = master.run_command(['ipa', 'group-add', 'groupc'])
+        assert 'Added group "groupc"' in cmd_output.stdout_text
+
+        # add group members
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'groupb', '--groups', 'groupa'])
+        assert 'Group name: groupb' in cmd_output.stdout_text
+        assert 'Member groups: groupa' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'groupc', '--groups', 'groupb'])
+        assert 'Group name: groupc' in cmd_output.stdout_text
+        assert 'Member groups: groupb' in cmd_output.stdout_text
+        assert 'Indirect Member groups: groupa' in cmd_output.stdout_text
+
+        # add user to group 'groupa'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'groupa', '--users', self.username])
+        assert 'Group name: groupa' in cmd_output.stdout_text
+        assert_str = 'Member users: {}'.format(self.username)
+        assert assert_str in cmd_output.stdout_text
+        assert 'Member of groups: groupb' in cmd_output.stdout_text
+        assert 'Indirect Member of group: groupc' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        # at this point, testuser001 has the following group memberships
+        # Member of groups: groupa, ipausers
+        # Indirect Member of group: groupb, groupc
+        cmd_output = master.run_command(['ipa', 'user-show', self.username])
+        assert 'groupa' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'groupb' in cmd_output.stdout_text
+        assert 'groupc' in cmd_output.stdout_text
+
+        clear_sssd_cache(client)
+
+        try:
+            cmd = ['ssh', '-i', '/tmp/user_ssh_priv_key',
+                   '-q', '{}@{}'.format(self.username, client.hostname),
+                   'groups']
+            cmd_output = master.run_command(cmd)
+            assert self.username in cmd_output.stdout_text
+            assert 'groupa' in cmd_output.stdout_text
+            assert 'groupb' in cmd_output.stdout_text
+            assert 'groupc' in cmd_output.stdout_text
+        except Exception as e:
```

Comment:
The try/except clause doesn't have much value here, you can simplify the code and remove it. If master.run_command fails, the test will fail anyway.

Reply:
fixed it